packages/npm/todo/src/operation-handlers/todo.ts

```diff
@@ -0,0 +1,11 @@
+import * as api from "todo-api";
+
+import { CommandHandlers } from "../commandHandlers/commandHandlers";
+import CreateTodo from "../commands/createTodo";
+
+export const addTodoOperationHandler: api.server.AddTodoItemOperationHandler<{}> = async (todo: string) => {
+    const createTodoCommand = new CreateTodo(todo);
+    const commandHandlers = new CommandHandlers();
```

Comment:
At some point this needs to be scoped to the server lifetime. So every server should have one commandHandlers object. There are various options to do this, like:
- use a global (absolutely not preferred!!)
- a hook (very "cool" and clean solution)
- create operationhandler in a factory, pass the command handlers to the factory

Let's discuss those options later! After implenting all the operationhandlers with dummy / example code